src/felles-komponenter/EkspanderbartTekstomrade.tsx

```diff
@@ -0,0 +1,45 @@
+import { NedChevron, OppChevron } from 'nav-frontend-chevron';
+import React, { useState } from 'react';
+
+import styles from './EkspanderbartTekstomrade.module.less';
+
+const MoreOrLess = ({ text, maxCharacters }: { text: string; maxCharacters: number }) => {
+    const [hasLongText, setHasLongText] = useState(true);
```

Comment:
visLangTekst?

Reply:
Nei, jeg synes ikke det all den tid dette a) er internt i komponenten, og b) ikke berører noe av domenespråket til NAV.

Eksternt framstår komponenten som "norsk", og det er bevisst for å holde seg til det jeg oppfatter er standarden, men som jeg riktig nok er uenig i.

Reply:
Er ikke egentlig det jeg sikter til.
Has longtekst virker ikke veldig som om den skal vise lang teks eller ikke.
Virker mer som om den har det i det hele tat.
Show longText?

Reply:
Litt nitpick dette? Brukes kun internt, men okei. Har pusha endring.